docs/ContributionGuidelines.md

```diff
@@ -0,0 +1,27 @@
+# Contribution Guidelines
+
+1. **Contributor**: An issue has to be created with a problem description and possible solutions using the github template. The better the problem and solution are described, the easier for the **Curators and Others** to understand it and the faster the process. In case of a bug, steps to reproduce will help a lot.
+2. **Curators and Others**: The curators will engage in a discussion about the problem and the possible solution. Others can join the discussion to bring other solutions and insights at any point.
```

Comment:
should we add what we discussed ? That by being a consistent contributor you become a curator ?

Reply:
Would you like to add this somewhere? Probably in a different section in the guidelines.